pkg/operator/ceph/file/mds/spec.go

```diff
@@ -202,3 +203,34 @@ func scaleMdsDeployment(ctx context.Context, clusterdContext *clusterd.Context,
 	}
 	return nil
 }
+
+// configureLivenessProbe returns the desired liveness probe for a given daemon
+func configureLivenessProbe(container *v1.Container, healthCheck cephv1.MirrorHealthCheckSpec) {
+	if ok := healthCheck.LivenessProbe; ok != nil {
```

Comment:
Yes, the great idea this will also benefit rgw.

So according to me the function would look like `func ConfigureStartupProbe(daemon cephv1.KeyType, container v1.Container) v1.Container {}`, but then 3 if-else would be needed for 3 health clusters,

Can you plz suggest any better way?

Reply:
Why aren't we using `ConfigureXProbe` from `github.com/rook/rook/pkg/operator/ceph/config`?